commands.go

```diff
@@ -206,6 +211,11 @@ func runCommands(line string, u *user, isSlack bool) {
 		return
 	}
 	if strings.HasPrefix(line, "./room") && !isSlack {
+		if u.messaging != nil {
+			u.messaging = nil
+			u.writeln(devbot, fmt.Sprintf("Left private chat and put you back in %s", u.room.name))
```

Comment:
Make this only say "left private chat":
```suggestion
			u.writeln(devbot, "Left private chat")
```
So that if a user is further deciding to join another room or dm someone else it won't say inconsistent stuff.